src/Components/Home.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const Home: React.FC = () => <p className="tabs__home">Home Page</p>;
```

Comment:
```suggestion
export const Home: React.FC = () => (
  <p className="tabs__home">Home Page</p>
);
```

short file is good, but clear code is even better

Reply:
it was VS code formatting)